WordPress/gutenberg
# Storybook: Add Story for EditableText component

Part of https://github.com/WordPress/gutenberg/issues/67165 

## What?

This PR will add story for `editable-text` component in the Storybook.

## Testing Instructions




1. Run npm run storybook:dev
2. Open the storybook on [localhost](http://localhost:50240/)
3. Check the `EditableText ` story.

## Screenshots or screencast 


<img width="1440" alt="Screenshot 2024-12-26 at 4 58 10 PM" src="https://github.com/user-attachments/assets/03e39976-1648-4d9c-8779-71441dcc464c" />
